satpy/scene.py

```diff
@@ -1022,6 +1029,71 @@ def to_geoviews(self, gvtype=None, datasets=None, kdims=None, vdims=None, dynami
 
         return gview
 
+    def to_hvplot(self, datasets=None, *args, **kwargs):
+        """Convert satpy Scene to Hvplot.
+
+        Args:
+            datasets (list): Limit included products to these datasets.
+            kwargs: hvplot options dictionary.
+
+        Returns: hvplot object that contains within it the plots of datasets list.
+                 As default it contains all Scene datasets plots and a plot title is shown.
+
+        Example usage::
+
+           scene_list = ['ash','IR_108']
+           plot = scn.to_hvplot(datasets=scene_list)
+
+           plot.ash+plot.IR_108
```

Comment:
This isn't enough to show the plot, what else needs to be done? I was testing this in IPython command line prompt.

Reply:
![Schermata del 2023-03-20 21-05-08](https://user-images.githubusercontent.com/12550360/226453473-76bc3504-a5ed-4802-805b-255fa95a48a5.png)

Reply:
@pnuu does it work in your Jupyter Notebook?